tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-docker-release-pipeline.yml

```diff
@@ -0,0 +1,74 @@
+parameters:
+- name: commit
+  displayName: Commit (Default to master or use sha1)
+  type: string
+  default: master
+- name: image_tag
+  displayName: Image Tag
+  type: string
+  default: latest
+
+variables:
+  docker_image_prefix: onnxruntime-training
+  linux_gpu_dockerfile: dockerfiles/Dockerfile.training
+
+name: $(Date:yyyyMMdd)$(Rev:.r)
+jobs:
+- job: Linux_py_GPU_Build_Test_Release_Dockerfile
+  timeoutInMinutes: 90
+  workspace:
+    clean: all
+  pool: Linux-GPU-CUDA10
+  steps:
+  - task: CmdLine@2
+    displayName: Build builder stage of docker file
+    inputs:
+      script: |
+        docker build \
+          --pull \
+          -t ${{ variables.docker_image_prefix }}-manylinux-gpu-release-stage1 \
+          --target builder \
+          --build-arg COMMIT="${{ parameters.commit }}" \
+          -f ${{ variables.linux_gpu_dockerfile }} .
+      workingDirectory: $(Build.SourcesDirectory)
+
+  - task: CmdLine@2
+    displayName: Run tests
+    inputs:
+      script: |
+        docker run \
+          --gpus all \
+          --rm \
+          ${{ variables.docker_image_prefix }}-manylinux-gpu-release-stage1 \
+          python onnxruntime/tools/ci_build/build.py \
+          --build_dir onnxruntime/build \
+          --config Release \
```

Comment:
since the configuration is specified here, perhaps BUILD_VERSION build arg should be set to the same value. that could be another variable or parameter.